### PR TITLE
Polish patch release documentation

### PR DESCRIPTION
**Pull request by asmacdo**
Signed-off-by: austin <austin@redhat.com>

### CODE REVIEW

**Comment by asmacdo** [comment]
These changes are based on my notes from last release. I'll vet them as I do the release today and remove the WIP label afterwards.

**Review by estroz — APPROVED** [review]
/lgtm

File: website/content/en/docs/contribution-guidelines/releasing.md [hunk 1]
```diff
@@ -160,35 +154,54 @@ git pull
...
-Commit these changes and push:
+All of the following changes should be present (and no others).
+
+1. `Makefile`: IMAGE_VERSION modified
+1. changelog/: 
+  1. all fragments deleted
+  1. changelog/generated/v1.3.1.md
+1. docs: 
+    add: `website/content/en/docs/upgrading-sdk-version/v1.3.1.md` (only if there are migration steps)
+    modified: installation docs link update. Is this still broken?TODO(asmacdo)
```
Comment (estroz) [inline website/content/en/docs/contribution-guidelines/releasing.md:187]: Nope all fixed (although the script is broken and [removes the `/download` path element](https://github.com/operator-framework/operator-sdk/blob/master/website/scripts/update_download_url.sh#L13), feel free to fix that in this PR).
```suggestion
    modified: installation docs link update.
```

File: website/content/en/docs/contribution-guidelines/releasing.md [hunk 2]
```diff
@@ -160,35 +154,54 @@ git pull
...
 ```
 
-### 2. Create and merge a new PR
+#### 3. Create and merge Pull Request
 
-Create and merge a new PR for the commit created in step 1. You can force-merge your PR to the locked-down `v1.3.x`
-if you have admin access to the operator-sdk repo, or ask an administrator to do so.
+1. Create and pull request against the `v1.3.x` branch. 
+2. Once 2 approving reviews are given, unlock the branch by setting
+"required approving reviewers" to back to 1. (See step 0).
```
Comment (asmacdo) [inline website/content/en/docs/contribution-guidelines/releasing.md:202]: "and" 
3. Merge

**Comment by jmrodri** [comment]
@asmacdo we should update the script so we don't have to fix the download url manually https://github.com/operator-framework/operator-sdk/blob/master/website/scripts/update_download_url.sh#L13

**Review by varshaprasad96 — APPROVED** [review]
since we are on this, another frequent error which I noticed is sometimes not using `gnu-sed` and `gnu-make`. Can we also add in the pre-req section, to verify if gnu sed and make commands are installed properly. Just by verifying that the output of `sed --version` and `make --version` are from gnu utilities. This happens frequently in Mac, where its installed but not linked.

**Comment by asmacdo** [comment]
@varshaprasad96 Sounds reasonable to me. As a non-mac user, I don't know exactly what output you should be looking for. Would you be willing to file an issue for that?

**Comment by varshaprasad96** [comment]
@asmacdo yes, sure.. will do that

**Review by rashmigottipati — APPROVED** [review]
/lgtm

Good fixes! The instructions are more clear now; now I know I would definitely ensure to pull down the `v1.y.x` branch and tag it :)

**Comment by estroz** [comment]
Links should be updated:
```
Old URL: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg
New URL: https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg
```

**Pull request merged**